### PR TITLE
master: OADP-2377 Add mustgather registry image to relatedImage

### PR DESCRIPTION
**Pull request by kaovilai**

### CODE REVIEW

**Review by rayfordj — CHANGES_REQUESTED** [review]
See review/comments on https://github.com/openshift/oadp-operator/pull/1136

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -809,6 +809,8 @@ spec:
                   value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
                 - name: RELATED_IMAGE_DATA_MOVER_DUMMY_POD
                   value: quay.io/konveyor/rsync-transfer:latest
+                - name: RELATED_IMAGE_MUSTGATHER
+                  value: registry.redhat.io/oadp/oadp-mustgather-rhel8
```
Comment (rayfordj) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:813]:
```suggestion
                  value: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
```

File: bundle/manifests/oadp-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -937,4 +939,6 @@ spec:
     name: kubevirt-velero-plugin
   - image: quay.io/konveyor/rsync-transfer:latest
     name: data-mover-dummy-pod
+  - image: registry.redhat.io/oadp/oadp-mustgather-rhel8
```
Comment (rayfordj) [inline bundle/manifests/oadp-operator.clusterserviceversion.yaml:942]:
```suggestion
  - image: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
```

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -58,6 +58,8 @@ spec:
             value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
           - name: RELATED_IMAGE_DATA_MOVER_DUMMY_POD
             value: quay.io/konveyor/rsync-transfer:latest
+          - name: RELATED_IMAGE_MUSTGATHER
+            value: registry.redhat.io/oadp/oadp-mustgather-rhel8
```
Comment (rayfordj) [inline config/manager/manager.yaml:62]:
```suggestion
            value: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
```

**Review by rayfordj — CHANGES_REQUESTED** [review]

**Review by rayfordj — APPROVED** [review]
/lgtm

**Review by sseago — APPROVED** [review]

**Pull request merged**